### DISCUSSION TITLE
Problem on dedicated server

### DISCUSSION THREAD

**Discussion by korda2310** [Vdauphin/HeartsAndMinds#1366] [Question and answer]
Is this mission also made to be on a dedicated server? Because I have a problem, I can't join the game after choosing my slot, I spawn as a seagull zeus (this is not a joke) how to remedy this problem? On my computer in single player I don't have this problem, everything works fine.
![20220501004132_1](https://user-images.githubusercontent.com/104427562/166125380-04d8666b-97ef-4212-8f9e-271aa9924020.jpg)

**Comment by Vdauphin** [comment]
Hello,

> Is this mission also made to be on a dedicated server? 

Yes

> how to remedy this problem? 

Check your .rpt for error and correct them until they disappear

>  On my computer in single player I don't have this problem, everything works fine.

Just fix your dedicated server

Cheers

**Reply by korda2310** [reply]
Thank you for your answer, the problem is that I have no error message via the dedicated server, because I can launch the mission normally, the problem is that I spawn as a seagull on any slot. I tested our other missions right after and everything works normally. I think the problem is the spawn system, I already mentioned it on another post that I would like to change this system. The only thing that changes from our other missions is the headless but I don't think that's where the problem comes from?

I really think that the problem comes from this spawn system that the server may not recognize? I launch via TADST

**Reply by Vdauphin** [reply]
I don't know It is working for everyone else

**Reply by korda2310** [reply]
I am not the only one https://github.com/Vdauphin/HeartsAndMinds/discussions/1345#discussion-3981422

**Reply by Vdauphin** [reply]
Base on [that](https://www.battlemetrics.com/servers/arma3?q=hearts%2Band%2Bminds%20||%20%22H%26M%22&status=online&sort=-players) you guys are

**Comment by SergDLeon** [comment]
Have you solved this problem?
